other/majority_vote_algorithm.py

```diff
@@ -0,0 +1,28 @@
+import collections
+
+"""
+This is Booyer-Moore Majority Vote Algorithm. The problem statement goes like Given an integer array of size n, find all elements that appear more than ⌊ n/k ⌋ times.
```

Comment:
One of the ❌ below is because line in Python code should not be longer than 88 characters.
```suggestion
This is Booyer-Moore Majority Vote Algorithm. The problem statement goes like:
Given an integer array of size n, find all elements that appear more than ⌊ n/k ⌋ times.
```